tests/c_languages/CPPCheckBearTest.py

```diff
@@ -14,22 +15,36 @@ def get_absolute_test_path(file):
 
 
 @generate_skip_decorator(CPPCheckBear)
-class CPPCheckBearTest(LocalBearTestHelper):
+class CPPCheckBearTest(unittest.TestCase):
 
     def setUp(self):
         self.section = Section('cppcheck')
-        self.uut = CPPCheckBear(self.section, Queue())
-        self.good_file = get_absolute_test_path('good_file.cpp')
-        self.bad_file = get_absolute_test_path('bad_file.cpp')
-        self.warn_file = get_absolute_test_path('warn_file.cpp')
-
-    def test_default(self):
-        self.check_validity(self.uut, [], self.good_file)
-        self.check_invalidity(self.uut, [], self.bad_file)
-        self.check_validity(self.uut, [], self.warn_file)
-
-    def test_enable(self):
         self.section.append(Setting('enable', 'unusedFunction'))
-        self.check_validity(self.uut, [], self.good_file)
-        self.check_invalidity(self.uut, [], self.bad_file)
-        self.check_invalidity(self.uut, [], self.warn_file)
+        self.file_dict = {}
+        self.queue = Queue()
+        self.test_files = ['good_file.cpp', 'bad_file.c', 'warn_file.cpp']
```

Comment:
Don't you want to not aggregate them here, so you can individually test files and make tests more reliable?

Reply:
@Makman2 here we don't have to test individual files, this is a `Globalbear ` and we can/should pass all the files at once.Shouldn't we?